cmd/create/cluster/cmd.go

```diff
@@ -1145,6 +1172,69 @@ func run(cmd *cobra.Command, _ []string) {
 			}
 		}
 	}
+	oidcEndpointUrl := args.oidcEndpointUrl
+	oidcPrivateKeySecretArn := args.oidcPrivateKeySecretArn
+	isByoOidcSet := false
+	if isSTS {
+		// Support byo oidc config
+		if oidcEndpointUrl != "" && interactive.Enabled() {
+			oidcEndpointUrl, err = interactive.GetString(
+				interactive.Input{
+					Question:   "OIDC Endpoint URL",
+					Help:       cmd.Flags().Lookup(OidcEndpointUrlFlag).Usage,
+					Required:   false,
+					Default:    oidcEndpointUrl,
+					Validators: []interactive.Validator{interactive.IsURL},
+				})
+			if err != nil {
+				r.Reporter.Errorf("Expected a valid OIDC Endpoint Url: %s", err)
+				os.Exit(1)
+			}
+			if oidcEndpointUrl != "" {
+				oidcPrivateKeySecretArn, err = interactive.GetString(
+					interactive.Input{
+						Question: "OIDC Private Key Secret ARN",
+						Help:     cmd.Flags().Lookup(OidcPrivateKeySecretArnFlag).Usage,
+						Required: true,
+						Default:  oidcPrivateKeySecretArn,
+					})
+				if err != nil {
+					r.Reporter.Errorf("Expected a valid ARN to the secret containing the private key: %s", err)
+					os.Exit(1)
+				}
+			}
+		}
+		isByoOidcSet = oidcEndpointUrl != "" || oidcPrivateKeySecretArn != ""
+		if isByoOidcSet {
+			attributesForByoOidc := []string{}
```

Comment:
NP
Maybe worth emphasizing this
```suggestion
			missingAttributesForByoOidc := []string{}
```